apps/aecore/src/aec_txs_trees.erl

```diff
@@ -38,13 +38,9 @@ root_hash(TxsTree) ->
 %%% Internal functions
 %%%===================================================================
 
-empty() ->
-    aeu_mtrees:empty().
-
-enter(K, V, T) ->
-    aeu_mtrees:enter(K, V, T).
-
-enter_signed_tx(SignedTx, TxsTree) ->
-    V = aetx_sign:serialize_to_binary(SignedTx),
-    K = aec_hash:hash(signed_tx, V),
-    enter(K, V, TxsTree).
+from_txs([],_Position, Tree) ->
+    Tree;
+from_txs([SignedTx|Left], Position, Tree) ->
+    Key = binary:encode_unsigned(Position),
+    Val = aetx_sign:serialize_to_binary(SignedTx),
```

Comment:
(Not in scope for this PR.)
Are signatures in the signed tx - if multiple signature - guaranteed to be sorted?

Reply:
I'm addressing this in the new serialization format.